src/nrc/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -235,220 +261,270 @@ msgstr "Succes"
 "created"
 msgstr ""
 
-#: datamodel/models.py:207
-msgid "datacontent"
+#: datamodel/models.py:217
+msgid "data content type"
 msgstr ""
 
-#: datamodel/models.py:211
+#: datamodel/models.py:221
 msgid ""
 "content type of the (optional) cloudevent data. Example: application/json"
 msgstr ""
 
-#: datamodel/models.py:215
+#: datamodel/models.py:225
 msgid "dataschema"
 msgstr ""
 
-#: datamodel/models.py:218
+#: datamodel/models.py:228
 msgid "A schema that the data must follow"
 msgstr ""
 
-#: datamodel/models.py:221
+#: datamodel/models.py:231
 msgid "subject"
 msgstr ""
 
-#: datamodel/models.py:224
+#: datamodel/models.py:234
 msgid "the events subject. Example: an uuid of a zaak"
 msgstr ""
 
-#: datamodel/models.py:227
+#: datamodel/models.py:237
 msgid "time"
 msgstr ""
 
-#: datamodel/models.py:230
+#: datamodel/models.py:240
 msgid "the timestamp of when the event happened"
 msgstr ""
 
-#: datamodel/models.py:233
+#: datamodel/models.py:243
 msgid "data"
 msgstr ""
 
-#: datamodel/models.py:235
+#: datamodel/models.py:246
 msgid "extra data using the format defined in datacontentype"
 msgstr ""
 
-#: datamodel/models.py:251
-msgid "substring"
+#: datamodel/models.py:268
+msgid "type substring"
 msgstr ""
 
-#: datamodel/models.py:254
+#: datamodel/models.py:271
 msgid "a substring of an event type that the subscription will watch for"
 msgstr ""
 
-#: templates/admin/base_site.html:22
-msgid "Ingelogd als"
-msgstr ""
+#: templates/403.html:13
+msgid "Sorry, you don't have access to this page (403)"
+msgstr "Sorry Je hebt geen toegang tot deze pagina (403)"
 
-#: templates/admin/base_site.html:31
-msgid "Change password"
-msgstr "Wachtwoord wijzigen"
+#: templates/404.html:13
+msgid "Sorry, the requested page could not be found (404)"
+msgstr "Sorry, the gevraagde pagina is niet gevonden (404)"
+
+#: templates/admin/base_site.html:5 templates/admin/base_site.html:20
+msgid "Administration"
+msgstr "Administratie"
 
 #: templates/admin/base_site.html:35
-msgid "Manage two-factor auth"
-msgstr "Tweefactorauthenticatie beheren"
+msgid "View site"
+msgstr "Bekijk site"
+
+#: templates/admin/base_site.html:41
+msgid "Account security"
+msgstr "Account beveiliging"
+
+#: templates/admin/base_site.html:44
+msgid "Change password"
+msgstr "Wachtwoord wijzigen"
 
-#: templates/admin/base_site.html:39
+#: templates/admin/base_site.html:49
 msgid "Log out"
 msgstr "Uitloggen"
 
 #: templates/admin/datamodel/abonnement/change_list.html:6
 msgid "Check status of callback URLs"
 msgstr "Haal status op van callback URLs"
 
+#: templates/includes/footer.html:16 templates/includes/footer.html:27
+#, fuzzy
+#| msgid "Documentation link"
+msgid "Documentation"
+msgstr "Documentatie link"
+
+#: templates/includes/footer.html:21 templates/includes/footer.html:37
+msgid "Code on Github"
+msgstr ""
+
+#: templates/includes/footer.html:32
+msgid "Docker image"
+msgstr ""
+
+#: templates/includes/footer.html:47
+msgid "Zaakgericht werken volgen VNG"
```

Comment:
I included a fix for this in https://github.com/open-zaak/open-zaak/pull/2254/files#diff-b3faf9abe9afbc897c9efded8918e6eaacca517e3b1fda3e0001154ffdf06f3bR38, can you include the same here + the translation?